helpers.js

```diff
@@ -63,6 +63,7 @@ const chooseDirection = function(closestMealArray, info) {
 };
 
 module.exports = {
+  getOpenSquares,
```

Comment:
This line needs to be deleted. getOpenSquares isn't defined above, so the server won't start.